### PR TITLE
Icon for commuter post info

### PR DESCRIPTION
**Pull request by mjoyce91**
- Adds icons for commuter post
- Adds commuter post frequency to results cards

Use with https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/374

Screen Shot 2020-06-12 at 4 17 33 PM

### CODE REVIEW

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 1]
```diff
@@ -186,6 +190,7 @@ class ResultsCard extends Component {
     if (isProjectedVacancy) cardClassArray.push('results-card--secondary');
     if (isTandem) cardClassArray.push('results-card--tandem');
     if (isTandem2) cardClassArray.push('results-card--tandem-two');
+    if (commuterPost) cardClassArray.push('results-card--commuterpost');
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCard/ResultsCard.jsx:193]: @mjoyce91 is this a placeholder for future?

**Comment by elizabeth-jimenez** [comment]
Resolution Checks (filter, pill, and card checked):
- [ ] 1280*800
- [ ] 1280*1024
- [ ] 1366*768
- [ ] 1920*1080

File: src/sass/_results.scss [hunk 2]
```diff
@@ -67,6 +67,23 @@ $results-container-width: 100% - $filter-container-width;
     }
   }
 
+  .commuter-banner {
+    height: 35px;
+    left: 0;
```
Comment (elizabeth-jimenez) [inline src/sass/_results.scss:72]: At some resolutions, see 1366*768, this looks better with `left: 1px`, but that's just me being nit-picky!

File: src/Components/ResultsCard/ResultsCard.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/ResultsCard/ResultsCard.jsx]: Good point, just removed

File: src/sass/_results.scss (hunk 2 above)
Comment (mjoyce91) [reply src/sass/_results.scss:72]: Hmm I'm not seeing a difference. I followed the designs and it has just a tiny bit of white space

Comment (elizabeth-jimenez) [reply src/sass/_results.scss:72]: Dang! Shoulda taken a SS. at that rez, on my screen, it almost looked like the top pixel of the border-left on the card expanded. But i am getting old 👓

Comment (elizabeth-jimenez) [reply src/sass/_results.scss:72]: image
Ok, got one! But honestly, you should be punished for using that resolution 🙃

**Pull request merged**